screen.css

```diff
@@ -0,0 +1,41 @@
+.table{
+    border:4px solid blue;
+    width:100%;
+    height:100%;
+    font-size:50px;
+  }
+  .first_row td{
+    border:4px solid black;
+    height:12%;
+    width:49%;
+  }
+  img{
+    height:200px;
+    width:49%;
+  }
+  .Heading{
```

Comment:
spacing is also not good

Reply:
should I work on div aliment or on the heading part  of the table and or the table data part